app/views/items/_items_table.html.erb

```diff
@@ -9,20 +9,24 @@
 <table class="table table-hover table-striped">
   <thead>
     <tr>
-      <th class="col-xs-6">Description</th>
+      <th class="text-center">Description</th>
       <th class="text-center">Quantity In Stock</th>
       <th class="text-center">Quantity Requested</th>
-      <th></th>
+      <th class="text-center">SKU</th>
+      <th class="text-center">Price</th>
+      <th class="text-center">Actions</th>
     </tr>
   </thead>
 
   <tbody>
     <% items.each do |item| %>
       <tr>
-        <td class="col-xs-6"><%= item.description %></td>
```

Comment:
Ya pulled to start with and am making visaul adjustments at the end once all data is in